### PR TITLE
Add a test to check if team members are org members

### PR DESCRIPTION
**Pull request by nikhita**
This would be useful to fail early if someone who is added as a team maintainer or member is not an org member. See https://github.com/kubernetes/org/pull/346#pullrequestreview-191923294 for an example.

This PR also validates:

- a user is not mentioned as both an org admin and org member
- a user is not mentioned as both a team maintainer and member
- no duplicates exist in the list of team maintainers and members

/assign @cblecker @fejta

### CODE REVIEW

File: config/kubernetes-sigs/org.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ members:
 - animeshsingh
 - ant31
 - apelisse
-- atoms
+- Atoms
```
Comment (nikhita) [inline config/kubernetes-sigs/org.yaml]:
```
--- FAIL: TestAllOrgs (0.03s)
    --- FAIL: TestAllOrgs/kubernetes-sigs (0.00s)
        config_test.go:169: The following members of team kubespray-maintainers are not org members: Atoms
FAIL
```

File: config/config_test.go [hunk 2]
```diff
@@ -87,6 +87,28 @@ func isSorted(list []string) bool {
 	return sort.StringsAreSorted(items)
 }
 
+// testTeamMembers checks if team maintainers and members are org members.
+// TODO: also ensure that the list is sorted.
+func testTeamMembers(teams map[string]org.Team, orgMembers sets.String) []error {
```
[the remark this answers is not in the document]
Comment (nikhita) [reply config/config_test.go]: Done. Also added a check for duplicate names in the lists.

File: config/kubernetes-sigs/org.yaml (hunk 1 above)
Comment (nikhita) [reply config/kubernetes-sigs/org.yaml]: Can we export this as well?

**Comment by nikhita** [comment]
> I wonder if this config validation stuff should actually be functions in the prow/config/ package that we export, so that they're reusable.

+1. Can we also make sure that we are sorting before dumping all names? We would be sorted (sorry couldn't resist) once we sort all names and add a test for new ones in the CI, but it might prove useful to others who will use peribolos.

**Comment by cblecker** [comment]
/lgtm
/approve

Thanks so much @nikhita! 🎈

**Pull request merged**